nntrainer/tensor/tensor.h

```diff
@@ -231,16 +227,41 @@ class Tensor {
     Tensor(std::vector<std::decay<decltype(d)>::type>{d}, t_type){};
 #endif
 
+  /**
+   * @brief     Constructor of Tensor
+   * @param[in] d data for the Tensor. It needs to set format properly.
+   * @param[in] t_type Tensor Type
+   */
+  Tensor(std::vector<std::vector<std::vector<std::vector<uint8_t>>>> const &d,
```

Comment:
We might need a tensor constructer with zero and scale.

Reply:
What else do we need besides the constructor with zero and scale?
Do you expect other methods or functions to improve with the parameters?

Reply:
For now, Zero and Scale (their vector type may need to be supported) are the only two I can think of, but there are more. @djeong20, let's discuss it. I think this can be done in a later PR.

Reply:
Another think is that we need to implement the tensor operations like add, mul, etc.